src/Container/Container.php

```diff
@@ -115,8 +175,12 @@ public function getNew($key)
      *
      * @param string $key
      */
-    public function destroyInstance($key)
+    public function destroyInstance($key): void
```

Comment:
```suggestion
    public function destroyInstance(string $key): void
```

and phpdoc can be removed